centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -498,10 +498,8 @@ object Operations extends StrictLogging {
 
       override def run: IO[Unit] = {
         if (CentaurConfig.expectCarbonite) {
-          logger.info(s"Expecting carbonited status for $workflowId")
           eventuallyArchived().timeout(CentaurConfig.metadataConsistencyTimeout)
         } else {
-          logger.info(s"Not expecting carbonited status for $workflowId (because expectCarbonite is turned off)")
           IO.pure(())
```

Comment:
Would reverting the `Remove unused log option [force ci]` commit work?

Reply:
(ie have it turned off by default but allow people to opt-in if they need it?)

Reply:
@kshakir added in the ticket description:
> Lots of INFO centaur.test.Operations$ - Not expecting carbonited status for 4a0cf1d9-e799-4d66-821e-4dadedb84130 (because expectCarbonite is turned off) in the logs.

Reply:
> IMHO been quite useful

Useful in centaur testing? If not, we can just turn it off for centaur.